drivers/backup/utils/backup_utils.go

```diff
@@ -0,0 +1,277 @@
+package utils
+
+import (
+	"fmt"
+	snapv1 "github.com/kubernetes-incubator/external-storage/snapshot/pkg/apis/crd/v1"
+	apapi "github.com/libopenstorage/autopilot-api/pkg/apis/autopilot/v1alpha1"
+	storkapi "github.com/libopenstorage/stork/pkg/apis/stork/v1alpha1"
+	"github.com/portworx/sched-ops/k8s/core"
+	"github.com/portworx/torpedo/drivers/node/ssh"
+	"github.com/portworx/torpedo/drivers/scheduler/spec"
+	"github.com/portworx/torpedo/pkg/log"
+	"github.com/portworx/torpedo/tests"
+	monitoringv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	admissionregistrationv1 "k8s.io/api/admissionregistration/v1"
+	appsapi "k8s.io/api/apps/v1"
+	batchv1 "k8s.io/api/batch/v1"
+	batchv1beta1 "k8s.io/api/batch/v1beta1"
+	corev1 "k8s.io/api/core/v1"
+	netv1 "k8s.io/api/networking/v1"
+	networkingv1beta1 "k8s.io/api/networking/v1beta1"
+	policyv1beta1 "k8s.io/api/policy/v1beta1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	storageapi "k8s.io/api/storage/v1"
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	apiextensionsv1beta1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+	"os"
+	"runtime"
+	"strings"
+)
+
+const (
+	// GlobalTorpedoWorkDirectory is where the Torpedo is located
+	GlobalTorpedoWorkDirectory = "/go/src/github.com/portworx/"
+	// GlobalKubeconfigDirectory is where the kubeconfig files should be stored
+	GlobalKubeconfigDirectory = "/tmp"
+)
+
+const (
+	// DefaultConfigMapName is the default config-map that stores kubeconfigs data
+	DefaultConfigMapName = "kubeconfigs"
+	// DefaultConfigMapNamespace is the namespace of the DefaultConfigMapName
+	DefaultConfigMapNamespace = "default"
+)
+
+const (
+	// DefaultInClusterName is the default cluster where the Torpedo is running
+	DefaultInClusterName = "source-cluster"
+	// DefaultSourceClusterName is the default cluster where the PX-Backup is installed
+	DefaultSourceClusterName = "source-cluster"
+	// DefaultDestinationClusterName is the default cluster where applications are installed
+	DefaultDestinationClusterName = "destination-cluster"
```

Comment:
Please update the comments as per our discussion.

Reply:
Done.